scout/commands/update/genes.py

```diff
@@ -45,6 +42,11 @@
     help="What genome build should be used. If no choice update 37 and 38.",
 )
 @click.option("--api-key", help="Specify the api key")
+@click.option(
+    "--files",
+    type=click.Path(exists=True, dir_okay=True, file_okay=False),
+    help="Path to directory with resource files",
+)
 @with_appcontext
 def genes(build, api_key):
```

Comment:
Yes, I actually intended to save this for a later PR: I realised it was starting to get out of hand with the number of files touched. But it would be nice to do - could solve both a test coverage issue and would essentially also fix the download-before-update issue. I think I will revert it for now - time flies.